Marlin/configuration_store.cpp

```diff
@@ -1315,8 +1315,8 @@ void MarlinSettings::reset() {
         SERIAL_ECHOLNPGM("UBL_MESH_MAX_X     " STRINGIFY(UBL_MESH_MAX_X));
         SERIAL_ECHOLNPGM("UBL_MESH_MAX_Y     " STRINGIFY(UBL_MESH_MAX_Y));
 
-        SERIAL_ECHOLNPGM("MESH_X_DIST        " STRINGIFY(MESH_X_DIST));
-        SERIAL_ECHOLNPGM("MESH_Y_DIST        " STRINGIFY(MESH_Y_DIST));
+        SERIAL_ECHOLNPAIR("MESH_X_DIST        ", MESH_X_DIST);
+        SERIAL_ECHOLNPAIR("MESH_Y_DIST        ", MESH_Y_DIST);
```

Comment:
I've been wondering if this compiles to a smaller binary. Then you could do the same in a few other places and save a few extra bytes.